prebuilt_forms/includes/user.php

```diff
@@ -129,6 +129,10 @@ function apply_user_replacements($original) {
   );
   // Do basic replacements and trim the data.
   $text = str_replace($replace, $replaceWith, $original);
+  // Allow other modules to hook in.
+  if (function_exists('hostsite_invoke_alter_hooks')) {
+    hostsite_invoke_alter_hooks('iform_user_replacements', $text);

```

Comment:
Does this hook need to be added to iform for Drupal 7.

Reply:
This is currently only used for ABLE to allow it to handle scheme specific stuff, and this is D8. The conditional means that no change is required for D7